plugins/importexport/csv/CSVImportExportPlugin.inc.php

```diff
@@ -82,190 +82,414 @@ function executeCLI($scriptName, &$args) {
 
 		$filename = array_shift($args);
 		$username = array_shift($args);
+		$basePath = dirname($filename);
 
 		if (!$filename || !$username) {
 			$this->usage($scriptName);
-			exit();
+			exit(1);
 		}
 
-		if (!file_exists($filename)) {
-			echo __('plugins.importexport.csv.fileDoesNotExist', array('filename' => $filename)) . "\n";
-			exit();
+		$userDao = DAORegistry::getDAO('UserDAO'); /* @var $userDao UserDAO */
+
+		$user = $userDao->getByUsername($username);
+		if (!$user) {
+			echo __('plugins.importexport.csv.unknownUser', ['username' => $username]) . "\n";
+			exit(1);
+		}
+
+		$file = null;
+
+		try {
+			$file = new SplFileObject($filename, 'r');
+		}
+		catch (Exception $e) {
+			echo $e->getMessage();
+			exit(1);
 		}
 
-		$data = file($filename);
+		$file->setFlags(SplFileObject::READ_CSV);
+
+		if ($file->eof()) {
+			echo __('plugins.importexport.csv.fileDoesNotExist', ['filename' => $filename]) . "\n";
+			exit;
+		}
```

Comment:
This can be removed, it should be covered by the try/catch, also this locale key seems to not exist.